<?xml version="1.0"?>


<launch>
	<include file="$(find andbot)/launch/display.launch" />

	<include file="$(find hector_mapping)/launch/mapping_default.launch" />

	<include file="$(find rplidar_ros)/launch/rplidar.launch" />

	<node pkg="andbot" type="mybot_rplidar_laser_angular_bound_filter" name="mybot_rplidar_laser_angular_bound_filter" />
<!--
	<include file="$(find andbot)/launch/xv_11.launch" />
-->
<!--
	<node pkg="andbot" type="mybot_xv11_angular_bound_filter" name="mybot_xv11_angular_bound_filter" />
-->	
	<node pkg="andbot" type="mybot_base_controller_v1" name="mybot_base_controller_v1" />	

	<!-- run serial node for base mega -->	
	<node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    		<param name="port" value="/dev/ttyACM2"/>
    		<param name="baud" value="1000000"/>
  	</node>

</launch>
